kpi/deployment_backends/kc_access/shadow_models.py

```diff
@@ -646,6 +646,32 @@ def storage_path(self):
         return str(self.media_file)
 
 
+class ReadOnlyKobocatDailyXFormSubmissionCounter(ReadOnlyModel):
+    class Meta(ReadOnlyModel.Meta):
+        app_label = SHADOW_MODEL_APP_LABEL
```

Comment:
It's better to use the inheritance`ReadOnlyModel.Meta` to avoid duplicating the `app_label` if we don't need to override it, e.g:

```python
    class Meta(ReadOnlyModel.Meta):
        db_table = 'logger_monthlyxformsubmissioncounter'
        indexes = [
            models.Index(fields=('year', 'month', 'user'))
        ]
```